api/templates/emails/registration_text.html.twig

```diff
@@ -0,0 +1,8 @@
+Hello {{ user.firstname }},
```

Comment:
Why `_text`? That doesn't generate some HTML!

Reply:
Because it is for the text field